react/PageEditor.tsx

```diff
@@ -50,4 +59,4 @@ const PageEditor: ComponentWithCustomMessage = (props: PageEditorProps) => {
 
 PageEditor.getCustomMessages = () => messages
 
-export default PageEditor
+export default withRuntimeContext(PageEditor)
```

Comment:
An alternative to the HoC is using `useRuntime` hook (it exists 😛)

Reply:
Oh, cool! I tried searching for "useRuntimeContext" and nothing came up, so I thought I was imagining things, haha.